client/src/api/lyricsApiSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
```

Comment:
this is too similar to lyricsApi.js
change name to reflect that this function should load lyrics from server, not from api, or change the other file in a meaningful way